specification/appendix/examples/metadata_examples/deprecating_columns_example.md

```diff
@@ -0,0 +1,77 @@
+# Deprecating Columns
+
+## Scenario
+
+ACME has decided to deprecate columns prior to removal from their FOCUS data export. The column for deprecation is x_awesome_column3. The provider creates a new [Schema](#schema) object to represent the new schema, with a unique [SchemaId](#schemaid).
+
+## Supplied Metadata
+
+Metadata can be provided at a location such as `/FOCUS/metadata/schemas/schema-34567-abcde-34567-abcde-34567.json`.
+
+The updated schema related metadata could look like this:
+
+```json
+ {
+  "SchemaId": "34567-abcde-34567-abcde-34567",
+  "FocusVersion": "1.0",
+  "CreationDate": "2024-03-02T12:01:03.083z",
+  "ColumnDefinition": [
+    {
+      "ColumnName": "BillingAccountId",
+                "DataType": "STRING",
```

Comment:
nit: Is this implying that datatypes should be all caps? I would expect either "String" or "string". We should standardize this in a future update to make metadata more usable across providers.